data/lang/module-cargorun/en.json

```
@@ -223,6 +235,10 @@
     "description": "custom cargo",
     "message": "Explosives"
   },
+  "EXPLOSIVES_RISK_MESSAGE_1": {
+    "description": "Answer to question: Will I be in any danger?",
+    "message": "No. Well, it's explosives. Not unless you drop it."
```

```suggestion
    "message": "No. Well, it's explosives, just don't drop it."
```
(Sorry, I know I made you tack on that second sentence, but flows better if it's part of the previous sentence)